src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a href="#home" target="_blank">
+          <img src="moyo_1.png" alt="logo">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="blue is-active nav__link">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#sumsung" class="nav__link">Sumsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a href="#laptop&computers"
+              data-qa="hover"
+              class="blue nav__link">Laptops & Computers</a>
```

Comment:
Fix the code style. The `blue` class is redundant here. You should change style for all links on hover  
```suggestion
              <a 
                href="#laptop&computers"
                data-qa="hover"
                class="nav__link"
              >
                Laptops & Computers
              </a>
```